xml/maintenance_updates_admin.xml

```diff
@@ -33,139 +33,126 @@
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The secure mode of Messenger v2 protocol is no longer experimental with
+    this release. This mode is now the preferred mode of connection for
+    monitors.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    <option>osd_deep_scrub_large_omap_object_key_threshold</option> has been
+    lowered to detect an object with large number of omap keys more easily.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The &dashboard; now supports silencing &prometheus; notifications.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <bridgehead renderas="sect1">&cephname; 14.2.2 Point Release</bridgehead>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The <literal>no{up,down,in,out}</literal> related commands have been
+    revamped. There are now two ways to set the
+    <literal>no{up,down,in,out}</literal> flags: the old <command>ceph osd
+    [un]set <replaceable>FLAG</replaceable></command> command, which sets
+    cluster-wide flags; and the new <command>ceph osd [un]set-group
+    <replaceable>FLAGS</replaceable> <replaceable>WHO</replaceable></command>
+    command, which sets flags in batch at the granularity of any crush node, or
+    device class.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    <command>radosgw-admin</command> introduces two subcommands that allow the
+    managing of expire-stale objects that might be left behind after a bucket
+    reshard in earlier versions of &ogw;. One subcommand lists such objects and
+    the other deletes them.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Earlier &cephname; releases (14.2.1 and 14.2.0) have an issue where
+    deploying a single new &cephname; &bluestore; OSD on an upgraded cluster
+    (i.e. one that was originally deployed pre-&cephname;) breaks the pool
+    utilization statistics reported by <command>ceph df</command>. Until all
+    OSDs have been reprovisioned or updated (via <command>ceph-bluestore-tool
+    repair</command>), the pool statistics will show values that are lower than
+    the true value. This is resolved in 14.2.2, such that the cluster only
+    switches to using the more accurate per-pool stats after
+    <emphasis>all</emphasis> OSDs are 14.2.2 or later, are &blockstore;, and
+    have been updated via the repair function if they were created prior to
+    &cephname;.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The default value for <option>mon_crush_min_required_version</option> has
+    been changed from 'firefly' to 'hammer', which means the cluster will issue
+    a health warning if your CRUSH tunables are older than 'hammer'. There is
+    generally a small (but non-zero) amount of data that will move around by
+    making the switch to 'hammer' tunables.
+   </para>
+   <para>
+    If possible, we recommend that you set the oldest allowed client to
+    'hammer' or later. To display what the current oldest allowed client is,
```

Comment:
Okay there is a lot of 'hammer' - is there a reason why you did not put these in <literal> tags?

Reply:
These are code names for Ceph versions. Is there a reason to have them all in a tag? If yes, i'll change them throughout the whole file. I'll ask team-wide, it's a stylish thing.

Reply:
so..Hammer or Ceph Hammer is the way to go for code names then.

Reply:
WFM